### PR TITLE
Create documentation component

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Create documentation component

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/4025

### CODE REVIEW

File: scripts/prepare-react-odata.sh [hunk 1]
```diff
@@ -22,14 +22,14 @@ while test $# -gt 0; do
     esac
 done
 
-# prepare react-components
+# prepare react-odata
```
Comment (aerfio) [inline scripts/prepare-react-odata.sh]: are you sure?

File: scripts/prepare-docs-component.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# arguments
+TEMP_FOLDER=
```
Comment (aerfio) [inline scripts/prepare-docs-component.sh]: equals ?

File: components/docs-component/tslint.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
```
Comment (aerfio) [inline components/docs-component/tslint.json]: add tslint-react-hooks

File: components/docs-component/src/render-engines/markdown-components/HeadersNavigation.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { HeadersNavigation as HN } from '@kyma-project/documentation-component';
+import { RenderedHeader } from './RenderedHeader';
+import { postProcessingHeaders } from '../../helpers';
+import { HeadersNavigationsWrapper, StyledHeadersNavigation } from './styled';
+
+export const HeadersNavigation: React.FunctionComponent = () => (
+  <HeadersNavigationsWrapper>
+    <div>
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply components/docs-component/src/render-engines/markdown-components/HeadersNavigation.tsx]: yes for styling

File: scripts/prepare-react-odata.sh (hunk 1 above)
Comment (magicmatatjahu) [reply scripts/prepare-react-odata.sh]: yes 😄

**Comment by magicmatatjahu** [comment]
Resolved by https://github.com/kyma-project/console/commit/34b9d2048be8cbeb282623575cde1d3534feae1b

**Comment by magicmatatjahu** [comment]
Resolved by https://github.com/kyma-project/console/commit/34b9d2048be8cbeb282623575cde1d3534feae1b

**Pull request closed without merging**